### PR TITLE
Update default_theme_builders.py - More efficient importing

### PR DESCRIPTION
**Pull request by vmc-7645**
Slightly faster due to only having to import functions from os/os.path instead of the entire package. Ran with no errors on my system.

### CODE REVIEW

**Comment by YuriSizov** [comment]
Thanks for your contribution! I think if we do this, this should be done to all Python scripts used by the buildsystem, so the code style is consistent. Would you mind doing that?

**Comment by vmc-7645** [comment]
Sounds good! Will try to get to that asap.

**Comment by YuriSizov** [comment]
In future, you can update PRs instead of recreating them. Just (force) push new changes to the same branch 🙃

**Comment by aaronfranke** [comment]
What makes this more efficient? At least in C++, things like `std::` are preferred because it makes it clear what namespace something belongs to. Wouldn't this hide the fact that we're using `os.path`?

**Comment by KoBeWi** [comment]
From quick googling:
https://stackoverflow.com/questions/3591962/python-import-x-or-from-x-import-y-performance
tl;dr it's a premature optimization that's minimally faster only in certain situations.

**Comment by akien-mga** [comment]
Yeah I'm not sure it's beneficial to do this. This will force us to import each function we want to use explicitly, and this makes the code less readable as we're now mixing namespaces functions with our own definitions.
